server/sonar-server/src/main/java/org/sonar/server/permission/ws/AddGroupAction.java

```diff
@@ -67,15 +74,32 @@ public void define(WebService.NewController context) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
+    AddGroupWsRequest addGroupWsRequest = toAddGroupWsRequest(request);
+    doHandle(addGroupWsRequest);
+
+    response.noContent();
+  }
+
+  private void doHandle(AddGroupWsRequest request) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00112)